src/main/kotlin/com/workos/user_management/models/AuthenticationChallenge.kt

```diff
@@ -0,0 +1,31 @@
+package com.workos.user_management.models
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * An authentication challenge
+ *
+ * @param object should be "authentication_challenge".
```

Comment:
We document the `object` param here, but I don't see it in the declaration below. Same for several other objects.

Is there but derived somehow elsewhere, or are we missing some code to deserialize it?

Reply:
Removed those comments, not needed anymore.